docs/user/gcp/dns.md

```diff
@@ -3,8 +3,8 @@
 Google's Cloud DNS service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup
 for the cluster to the outside world. To use OpenShift, you must have created a public zone in Google Cloud DNS in
 the same project as your OpenShift cluster. You must also ensure the zone is "authoritative" for the domain. There are
-two ways to do this outlined below: root domain and subdomain. A root domain is `example.com`. A subdomain is of
-the form `clusters.example.com`.
+two ways to do this outlined below: root domain and subdomain. A root domain is `openshiftcorp.com`. A subdomain is of
```

Comment:
The reason I switched to openshiftcorp.com (the name used in AWS) was in order to create the screenshots. I cannot create zones in GCP with example.com because I get this  error:
> This domain (or its child domains) may be reserved or registered already, prohibited by policy, or there may be no DNS name server shards remaining for the name. For more information, see this documentation  

Please advise.

Reply:
the screenshots can have `openshiftcorp.com` for now, but the doc should have `example.com`